wasmsdk/demo/zcn.js

```diff
@@ -161,6 +161,7 @@ async function bulkUpload(options) {
       fileSize: obj.file.size,
       thumbnailBytes:obj.thumbnailBytes?obj.thumbnailBytes.toString():"",
       encrypt:obj.encrypt,
+      encrypt:obj.webstreaming,
```

Comment:
Should it be `webstreaming` here?

```suggestion
      webstreaming:obj.webstreaming,
```